variables.tf

```diff
@@ -95,4 +95,10 @@ variable "bamboo_admin_display_name" {
 variable "bamboo_admin_email_address" {
   description = "Bamboo system administrator email address."
   type        = string
+}
+
+variable "number_of_bamboo_agents" {
+  description = "Number of Bamboo remote agents."
```

Comment:
Should we validate for the value being `>= 0`? (if we allow 0, see comment above).

Reply:
Added.